### ISSUE TITLE
CrudField JavaScript Library - not able to set value on all field types

### ISSUE THREAD

**Issue by pxpm** [Laravel-Backpack/CRUD#5009]

**Comment by tabacitu** [comment]
I totally agree we should do this at some point... but it's A LOT of work to do this for all field types... so let's postpone this. Until then, developers who need to do this can find individual solutions for their use cases.

**Comment by tabacitu** [comment]
Good find! 👍
